### DISCUSSION TITLE
Don't you think that swiper 'autoplay' instance needs to have a parameter that can turn on or off the autoplay?

### DISCUSSION THREAD

**Discussion by rasibweb** [nolimits4web/swiper#7272] [General]
For example see this see this code
`
      var autoplay_speed = 2000;
      var pauseOnHover = true;
      var pauseOnInteractionr = false;
       autoplay: {
            delay: autoplay_speed,
            pauseOnMouseEnter: pauseOnHover,
            disableOnInteraction: pauseOnInteraction,
        },`
like this i can easily control swiper activation dinamically. But i can't add a functionality to turn off or on autoplay. if there was a parameter in autoplay instance like delay or pauseOnMouseEnter to turn off or on autoplay that would be awesome.

**Comment by 9romise** [comment]
Do you mean  [`swiper.autoplay.pause()`](https://swiperjs.com/swiper-api#method-swiper-pause) ?

**Reply by rasibweb** [reply]
No I mean this 

`autoplay {
         pauseOnHover : true,
         autoplayInit:true, // I'm taking about this parameter should be there
}`

**Reply by anobjectn** [reply]
swiper.autoplay.pause() is what I was looking for, but Im going to say the documentation is not clear on this, https://swiperjs.com/swiper-api#method-swiper-pause where its just swiper.pause()

**Reply by ddtbuilder** [reply]
not only is the documentation wrong....it doesn't even work. I've tried pause/resume/start/stop and none of them seem to have any affect at all.
